tests/js/session-tests.js

```diff
@@ -1,1593 +0,0 @@
-    let progressCalled = false;
-
-    const credentials = Realm.Credentials.anonymous();
-    let app = new Realm.App(appConfig);
-    return runOutOfProcess(
-      __dirname + "/download-api-helper.js",
-      appConfig.id,
-      appConfig.baseUrl,
-      partition,
-      REALM_MODULE_PATH,
-    )
-      .then(() => {
-        return app.logIn(credentials);
-      })
-      .then((user) => {
-        let config = getSyncConfiguration(user, partition);
-
-        return Promise.race([
-          Realm.open(config).progress((transferred, total) => {
-            progressCalled = true;
-          }),
-          new Promise((_, reject) =>
-            setTimeout(
-              () => reject("Progress Notifications API failed to call progress callback for Realm constructor"),
-              5000,
-            ),
-          ),
-        ]);
-      })
-      .then(() => TestCase.assertTrue(progressCalled));
-  },
-
-  // test that it is possible to register a custom logging function on a
-  // Sync session, and that it is invoked when a session is established
-  async testCustomSessionLogging() {
-    const partition = Utils.genPartition();
-    let credentials = Realm.Credentials.anonymous();
-    let app = new Realm.App(appConfig);
-    let user = await app.logIn(credentials);
-    let config = getSyncConfiguration(user, partition);
-
-    const logLevelStr = "info"; // "all", "trace", "debug", "detail", "info", "warn", "error", "fatal", "off"
-    const logLevelNum = 4; // == "info", see index.d.ts, logger.hpp for definitions
-
-    const promisedLog = new Promise((resolve) => {
-      Realm.App.Sync.setLogLevel(app, logLevelStr);
-      Realm.App.Sync.setLogger(app, (level, message) => {
-        if (level == logLevelNum && message.includes("Connection") && message.includes("Session")) {
-          // we should, at some point, receive a log message that looks like
-          // Connection[1]: Session[1]: client_reset_config = false, Realm exists = true, client reset = false
-          resolve(true);
-        }
-      });
-    });
-
-    const realm = await Realm.open(config);
-    await promisedLog;
-    realm.close();
-  },
-
-  testAddConnectionNotification() {
-    const partition = Utils.genPartition();
-    let app = new Realm.App(appConfig);
-    const credentials = Realm.Credentials.anonymous();
-    return app
-      .logIn(credentials)
-      .then((u) => {
-        let config = getSyncConfiguration(u, partition);
-        return Realm.open(config);
-      })
-      .then((realm) => {
-        return new Promise((resolve, reject) => {
-          realm.syncSession.addConnectionNotification((newState, oldState) => {
-            if (
-              oldState === Realm.App.Sync.ConnectionState.Connected &&
-              newState === Realm.App.Sync.ConnectionState.Disconnected
-            ) {
-              resolve();
-            }
-          });
-          realm.close();
-        });
-      });
-  },
```

Comment:
@JacobOscarGunnarsson this seems to be missing from the migrated tests.

Reply:
Or is it this? https://github.com/realm/realm-js/pull/5391/files#diff-0ab9adc1bbb7aee926da3074f9a75569f592434ba0585601cb9321ff5410a7d6R383-R404